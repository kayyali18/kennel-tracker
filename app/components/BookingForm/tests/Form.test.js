// import React from 'react'
// import renderer from 'react-test-renderer'
// import 'jest-styled-components'

// import Form from '../Form'

// describe('<Form />', () => {
//   it('should match snapshot', () => {
//     const renderedComponent = renderer.create(<Form />).toJSON()
//     expect(renderedComponent).toMatchSnapshot()
//   })

//   it('should contain a form inside the div', () => {
//     const renderedComponent = renderer.create(<Form />).toJSON()
//     expect(renderedComponent.type).toEqual('div')
//     expect(renderedComponent.children[0].type).toEqual('form')
//   })
// })
