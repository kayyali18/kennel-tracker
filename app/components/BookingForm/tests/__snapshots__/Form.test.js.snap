// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should match snapshot 1`] = `
.c0 {
  display: grid;
  grid-gap: 4rem;
  grid-template-areas: 'input .';
  height: 100%;
  position: absolute;
  overflow: scroll;
  width: 100%;
}

.c0 input {
  border: 2px solid black;
  position: relative;
  height: 50px;
}

.c0 input:nth-child(odd) {
  top: 2rem;
  left: 4rem;
}

.c0 input:nth-child(even) {
  top: 6rem;
  right: 4rem;
}

<div>
  <form
    className="c0"
  >
    <input
      placeholder="Name"
      type="text"
    />
    <input
      placeholder="Address"
      type="text"
    />
    <input
      placeholder="Cellphone"
      type="number"
    />
    <input
      placeholder="Homephone"
      type="number"
    />
    <input
      placeholder="E-mail"
      type="email"
    />
    <input
      placeholder="Emergency Contact Name"
      type="text"
    />
    <input
      placeholder="Emergency Contact #"
      type="number"
    />
  </form>
</div>
`;
